src/Table/Table.stories.tsx

```diff
@@ -29,6 +33,28 @@ storiesOf('Table', module).add(
         sortable={boolean('sortable', true)}
         multiSort={boolean('multiSort', true)}
       />
+      <h2>Checkbox with Table</h2>
```

Comment:
Let's make the checkbox table a separate Storybook story so that the generated code in the docs doesn't include both tables.